pages/guides/developing/encrypted-vod.en-US.mdx

```diff
@@ -101,7 +101,7 @@ Now, we can use asymmetric encryption with the Livepeer Public Key to encrypt ou
 
 ```js
 // Decode the SPKI public key from base64 and convert it to a buffer
-spkiPublicKey = atob(publicKeyData.spki_public_key);
+const spkiPublicKey = atob(publicKeyData.spki_public_key);
 const publicKeyBuffer = Uint8Array.from(atob(spkiPublicKey), (c) =>
```

Comment:
Isn't `spkiPublicKey` var already `atob`'ed? Do we have some double b64 encoding somewhere?

Reply:
Subtle crypto want the actual key atobed, so we need to atob it again